install-odoo-saas.sh

```diff
@@ -26,7 +26,7 @@
  export ODOO_DATA_DIR=${ODOO_DATA_DIR:-"/opt/odoo/data/"}
  export BACKUPS_DIR=${BACKUPS_DIR:-"/opt/odoo/backups/"}
  export LOGS_DIR=${LOGS_DIR:-"/var/log/odoo/"}
- export OPENERP_SERVER=${OPENERP_SERVER:-/etc/openerp-server.conf}
+ export ODOO_RC=${ODOO_RC:-/etc/odoo-server.conf}
```

Comment:
[It seems ok ](https://github.com/odoo/odoo/blob/11.0/odoo/tools/config.py#L385
)to use ODOO_RC.  
But, in that case we need to rename OPENERP_SERVER in other files (dockerfiles and some init scripts)

Reply:
In fact, it will be better but waited for the approval of the idea.
So I made similar changes request in the other files too.